src/main/java/net/sf/jabref/preferences/PreviewPreferences.java

```diff
@@ -0,0 +1,108 @@
+package net.sf.jabref.preferences;
+
+import java.util.List;
+import java.util.Map;
+
+import net.sf.jabref.logic.citationstyle.CitationStyle;
+
+import static net.sf.jabref.Globals.prefs;
+
+
+public class PreviewPreferences {
+
+    private static final String CYCLE_PREVIEW_POS = "cyclePreviewPos";
+    private static final String CYCLE_PREVIEW = "cyclePreview";
+    private static final String PREVIEW_PANEL_HEIGHT = "previewPanelHeight";
+    private static final String PREVIEW_STYLE = "previewStyle";
+    private static final String PREVIEW_ENABLED = "previewEnabled";
+
+    private final JabRefPreferences preferences;
+
+
+    public PreviewPreferences(JabRefPreferences preferences) {
```

Comment:
Reading isn't a Problem, but writing looks quite ugly in my opinion, or how else could I do it?

```Java
PreviewPreferences previewPreferences = preferences.getPreviewPreferences();
PreviewPreferences newPreviewPreferences = new PreviewPreferences(previewheight, previewPreferences.isEnabled(), previewPreferences.getpreviewStyle(), previewPreferences.getPreviewCycle());
```